doc/apiv3-documentation.apib

```diff
@@ -3155,6 +3233,7 @@ Note that due to sharing this might be more than the versions *defined* by that
 | category           | The category of the work package                                                                                                                      | Category     |              | READ / WRITE          |                                           |
 | priority           | The priority of the work package                                                                                                                      | Priority     | not null     | READ / WRITE          |                                           |
 | project            | The project to which the work package belongs                                                                                                         | Project      | not null     | READ                  |                                           |
+| revisions          | Revisions that are referencing the work package                                                                                                       | Revision     |              | READ                  | **Permission** view changesets            |
```

Comment:
DId you forget to specify the resource corresponding to this link? (e.g. `/api/v3/work_packages/42/revisions`)

Or do you plan to explicitly enumerate them inline? (I'd personally oppose doing that)

In any case you *should* add an example to the WP resource below ^^